cellrank/tl/_lineage.py

```diff
@@ -706,6 +706,44 @@ def copy(self, _="C") -> "Lineage":
     def __copy__(self):
         return self.copy()
 
+    def priming_degree(self, early_cells: Optional[np.ndarray] = None) -> np.ndarray:
+        """
+        TODO.
```

Comment:
Wait, why do we have this? I thought we went for something unified that can compute the lineage priming using either kl divergence or entropy?